doc/src/manual/arrays.md

```diff
@@ -72,7 +72,10 @@ or a series of dimension sizes passed as a variable number of arguments.
 
 [^1]: *iid*, independently and identically distributed.
 
-The syntax `[A, B, C, ...]` constructs a 1-d array (vector) of its arguments. If all arguments have a common promotion type then they get converted to that type. In such a situation mutable arguments are stored in the new array by reference only if they do not have to be converted.
+The syntax `[A, B, C, ...]` constructs a 1-d array (vector) of its arguments. If all
+arguments have a common promotion type then they get converted to that type. In such
+a situation mutable arguments are stored in the new array by reference only if they do
```

Comment:
this isn't really about being stored by reference - it's that `convert(T, x)` returns `x` if `x` is already of type `T`, but (usually) returns a separate object otherwise